### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by ethedev**

### CODE REVIEW

File: src/components/layouts/Navbar.tsx [hunk 1]
```diff
@@ -24,24 +25,27 @@ export const Navbar = () => {
...
-      <div className="flex-column expand padding-right-3 tablet-padding-x-4 tablet-padding-y-8 min-height-full">
+      <div className="flex-column expand padding-right-3 tablet-width-full tablet-padding-x-4 tablet-padding-y-8 min-height-full">
         <div className="margin-left-8 margin-top-10 tablet-margin-0 width-56 portrait-margin-0 landscape-margin-0"></div>
-        <NavbarButton text="Explore Synths" icon="Globe" to="/synths" />
+        <NavbarButton text="Explore Synths" icon="Globe" to="/explore" />
         <NavbarButton text="Portfolio" icon="User" to="/portfolio" />
         <div className="nav-divider margin-y-5"></div>
         <h6 className="margin-left-8 padding-left-3 tablet-padding-left-0 tablet-margin-left-3">Learn</h6>
-        <NavbarButton text="Docs" icon="Book" to="https://yam.gitbook.io/synths/" external />
+        <NavbarButton text="Docs" icon="Book" to="https://yam.gitbook.io/explore/" external />
```
Comment (ethedev) [inline src/components/layouts/Navbar.tsx]: @sali0 that link is broken, same for all the other docs links in this commit, and moving forward we need to fix and update the base of these links to this one instead **https://docs.synths.yam.xyz/**

File: src/pages/Landing.tsx [hunk 2]
```diff
@@ -100,22 +100,16 @@ export const Landing: React.FC = () => {
             <SynthBlock
               name="uPUNKS"
               image="cryptopunk"
-              url="/synths/uPUNKS"
+              url="/uPUNKS"
```
Comment (ethedev) [inline src/pages/Landing.tsx]: @sali0 this isnt working

Comment (ind-igo) [reply src/pages/Landing.tsx]: fixed

File: src/pages/Landing.tsx [hunk 3]
```diff
@@ -261,7 +256,7 @@ export const Landing: React.FC = () => {
               >
                 Blog
               </a>
-              <a href="https://yam.finance/" target="_blank" rel="noreferrer" className="block margin-bottom-2">
+              <a href="https://yam.xyz/" target="_blank" rel="noreferrer" className="block margin-bottom-2">
```
Comment (ethedev) [inline src/pages/Landing.tsx]: we still havent moved to the new website yet, soon we can update the link
```suggestion
              <a href="https://yam.finance/" target="_blank" rel="noreferrer" className="block margin-bottom-2">
```

**Pull request merged**